### DISCUSSION TITLE
Rethink the extension's accent color

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#5916] [Discussion]
Currently, our accent color is under the CSS variable "--brand-orange": `#ff7b26`. Talking contrast, this is not a good color to use. The contrast to the light-mode background and the text doesn't pass the minimum contrast ratio recommended by the WCAG guidelines (in the case of the text, it's 4.5 at a minimum, and 7 if we want enhanced contrast).

We need a color that has enough contrast for people to make it out against the background and the text on it, too, and it needs to on both light and dark mode.

Ideally, this new color would apply to on/off switches set to on, selected single-select options, selected tabs/categories and maybe the main header containing the Scratch Addons logo and probably change based on the theme you were using, so it would be brighter in dark mode and darker in light mode.

So, what should the new colors be? And should we change the text color too?

(From #5914) What do you think @WorldLanguages @mxmou?

**Comment by WorldLanguages** [comment]
Personally regarding the brand color (which is also used in the logo), I'm open to changing it to a completely different color. In other words, I don't think it necessarily needs to be a shade of orange, so we shouldn't restrict our brand color to that. It would be a pretty drastic change for our brand and users would notice it, but I think we could try to think outside the box, at least for a bit.
